pnpm-lock.yaml

```diff
@@ -357,7 +355,7 @@ importers:
       xterm-addon-web-links: 0.5.1_xterm@4.18.0
       xterm-addon-webgl: 0.11.4_xterm@4.18.0
     devDependencies:
-      '@senecacdot/eslint-config-telescope': 1.1.0_eslint@7.32.0
+      '@senecacdot/eslint-config-telescope': link:../../../tools/eslint
```

Comment:
Might be the same issue that David mentioned above? I'm not familiar with the latest stuff added since Satellite was recently incorporated